heft-plugins/heft-sass-plugin/src/SassTypingsPlugin.ts

```diff
@@ -21,38 +20,31 @@ const PLUGIN_NAME: string = 'SassTypingsPlugin';
 const PLUGIN_SCHEMA_PATH: string = `${__dirname}/schemas/heft-sass-plugin.schema.json`;
 const SASS_CONFIGURATION_LOCATION: string = 'config/sass.json';
 
-export class SassTypingsPlugin implements IHeftPlugin {
+export default class SassTypingsPlugin implements IHeftPlugin {
   private static _sassConfigurationLoader: ConfigurationFile<ISassConfigurationJson> | undefined;
 
-  public readonly pluginName: string = PLUGIN_NAME;
-  public readonly optionsSchema: JsonSchema = JsonSchema.fromFile(PLUGIN_SCHEMA_PATH);
-
   /**
    * Generate typings for Sass files before TypeScript compilation.
    */
-  public apply(heftSession: HeftSession, heftConfiguration: HeftConfiguration): void {
-    heftSession.hooks.build.tap(PLUGIN_NAME, (build: IBuildStageContext) => {
-      build.hooks.preCompile.tap(PLUGIN_NAME, (preCompile: IPreCompileSubstage) => {
-        preCompile.hooks.run.tapPromise(PLUGIN_NAME, async () => {
-          await this._runSassTypingsGeneratorAsync(
-            heftSession,
-            heftConfiguration,
-            build.properties.watchMode
-          );
-        });
-      });
+  public apply(taskSession: IHeftTaskSession, heftConfiguration: HeftConfiguration): void {
+    taskSession.hooks.clean.tapPromise(PLUGIN_NAME, async (cleanOptions: IHeftTaskCleanHookOptions) => {
+      // No-op. Currently relies on the TypeScript plugin to clean up the generated typings.
```

Comment:
Shouldn't this plugin own the output folder for sass typings, and the typescript plugin only reads from it?

Reply:
Yes, it should. I think ideally we should eventually use the stub files concept to introduce a lib-css folder, but that's down the road.